themes/classic/_dev/js/checkout.js

```diff
@@ -51,9 +51,21 @@ function setUpCheckout() {
   });
 }
 
+function toggleImage() {
+  $('.card-block .cart-summary-products p a').on('click', function (icon) {
+    icon = $(this).find('i.material-icons');
+    if (icon.text() == "expand_more") { 
```

Comment:
use quote everywhere please
```js
icon.text(icon.text() == 'expand_more' ? 'expand_less' : 'expand_more');
```

Reply:
I removed them

Reply:
Nope, look target line
```suggestion
    if (icon.text() == "expand_more") { 
```

Reply:
Okay, I did it. Normally there's none